src/config.js

```diff
@@ -51,6 +51,8 @@ export const BIOPAX_CONVERTER_URL = env('BIOPAX_CONVERTER_URL', 'https://biopax.
 export const GROUNDING_SEARCH_BASE_URL = env('GROUNDING_SEARCH_BASE_URL', 'https://grounding.baderlab.org');
 export const NCBI_EUTILS_BASE_URL = env('NCBI_EUTILS_BASE_URL', 'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/');
 export const NCBI_EUTILS_API_KEY = env('NCBI_EUTILS_API_KEY', 'b99e10ebe0f90d815a7a99f18403aab08008');
+export const INDRA_STATEMENTS_URL = env('INDRA_STATEMENTS_URL', 'https://db.indra.bio/latest/statements/from_agents');
```

Comment:
I understand but I have a single concern about that. Since these values are environment variables it may make things complicated to strip off that. Assume that I set another variable for ``https://db.indra.bio/latest/`` and let's name it as ``INDRA_DB_BASE_URL``. Then I will need to have another variable for the rest of string. I guess some of these values being dependent on the others may be undesired in some cases. However, I think both may have some advantadges/disadvantadges. What do you think?

Reply:
Just have one variable in config.js `INDRA_DB_BASE_URL` set to `https://db.indra.bio/latest/`. Then in the module just append `statements/from_agents` as required. 

Reason I suggest this is that I notice a bunch of potentially useful services on that domain. https://github.com/indralab/indra_db/tree/master/rest_api

Reply:
Okay

Reply:
The main motivation of variables like INDRA_DB_BASE_URL is just to make it so you can swap out the server that the app points to.  The intention isn't to make things completely general.